docs/DeveloperGuide.md

```diff
@@ -35,4 +213,21 @@
 
 ## Instructions for manual testing
```

Comment:
Maybe the manual testing section can have more commands to be more comprehensive (e.g delete, list, or edit)? Rather than testing each command in isolation, maybe can group them together for a single test(e.g show add first, then list, then delete, then list, etc......)?